requirements.in

```diff
@@ -3,6 +3,9 @@
 # File: https://raw.githubusercontent.com/apache/airflow/constraints-2.8.2/constraints-3.11.txt
 --constraint ./constraints.txt
 
+# Package management
+pip-tools
```

Comment:
pip-tools should not be part of the project requirements. This is a development requirement. In the current implementation, pip-tools will be included in the Docker image. Pip-tools is listed in the README.md section "Prerequesites". We could also create a dev-requirements file or some similar implementation.

Reply:
There are already dev requirements like pytest, Black, and Ruff in there though (some of which were [added by you](https://github.com/mozilla/telemetry-airflow/pull/1658/files#diff-de469cc9cf8992ffaec661b4e087cfded3ec6da722072ec1fdf11f61690fd1b8)), so I figured I was following the convention with this approach.

That said, I'm personally in favor of the dev requirements approach (including moving pytest, Black, and Ruff out of the main requirements), so I'll change tack and do that as a new PR and close this one.

Reply:
git blame strikes once again :laughing: You are 100%, moving ruff, pytest and others to `requirements-dev.txt` is the right approach.